pennylane_lightning/core/_measurements_base.py

```diff
@@ -56,6 +56,7 @@ def __init__(
     ) -> None:
         self._qubit_state = qubit_state
 
+        self._use_mpi = False
```

Comment:
Do we want to modify the base-class API to have a term that may not be used?
I think a subclass with the new term might be better. Or is there a reason we cannot do that?

Reply:
Sorry, this was a dirty solution for issues with the mpi inclusion. However, I have already opened a story to solve this issue with the base class [sc-74360]

Reply:
I was checking the code and I think this `use_mpi` should stay just in the measurements class because the serialize function needs this variable independently of the device.